python/tk_multi_publish/publish.py

```diff
@@ -361,23 +389,29 @@ def _scan_scene(self):
                 
         return items
         
-    def _do_pre_publish(self, primary_task, secondary_tasks, progress_cb):
+    def _do_pre_publish(self, primary_task, secondary_tasks, progress_cb, user_data):
         """
         Do pre-publish pass on tasks using the pre-publish hook
         """
         
         # do pre-publish of primary task:
-        primary_task.pre_publish_errors = self._app.execute_hook("hook_primary_pre_publish",  
-                                                                task=primary_task.as_dictionary(), 
-                                                                work_template = self._work_template,
-                                                                progress_cb=progress_cb)
+        primary_task.pre_publish_errors = self._app.execute_hook(
+            "hook_primary_pre_publish",  
+            task=primary_task.as_dictionary(), 
+            work_template=self._work_template,
+            progress_cb=progress_cb,
+            user_data=user_data,
```

Comment:
Spoke with Rob and we agree that we can mention it in the release notes and that should be enough. Since the hooks already accept **kwargs we can be fairly sure that most cases are covered.